src/Spd.Resource.Repository/ControllingMemberCrcApplication/Contract.cs

```diff
@@ -38,7 +38,11 @@ public record ControllingMemberCrcApplication
     public bool? IsTreatedForMHC { get; set; }
     public ResidentialAddr? ResidentialAddressData { get; set; }
     public IEnumerable<UploadedDocumentEnum>? UploadedDocumentEnums { get; set; }
-    public Guid BizContactId { get; set; } 
+    public Guid BizContactId { get; set; }
+    //TODO: check these fields
+    public Guid? ExpiredLicenceId { get; set; }
+    public bool? HasExpiredLicence { get; set; }
+    public LicenceTermEnum? LicenceTermCode { get; set; }
```

Comment:
these fields should come from fe req.
the field `HasExpiredLicence `is added and it is used by commit app function.

Reply:
really? I did not see it used in commit app

Reply:
keeping them for renewal application